app.js

```diff
@@ -1,8 +1,66 @@
+/* eslint-disable no-undef */
+/* eslint-disable no-unused-vars */
 // import functions and grab DOM elements
+const case1Button = document.getElementById('case1-button');
+const case2Button = document.getElementById('case2-button');
+const case3Button = document.getElementById('case3-button');
 
+const case1Container = document.getElementById('case-container');
+const case2Container = document.getElementById('case2-container');
+const case3Container = document.getElementById('case3-container');
+const results = document.getElementById('results');
+
+const totalEl = document.getElementById('total');
+const lossesEl = document.getElementById('losses');
+const winsEl = document.getElementById('wins');
+
+const cases = ['case', 'case2', 'case3'];
 // let state
+let totalScore = 0;
+let totalWins = 0;
 
 // set event listeners 
-  // get user input
-  // use user input to update state 
-  // update DOM to reflect the new state
+case1Button.addEventListener('click', () => {
+    const caseSelect = Math.floor(Math.random() * 3);
+    const answer = cases[caseSelect];
+    handleGuess(answer, 'case1');
```

Comment:
Pay close attention to the details here. You have your userGuess set to `case1`, but in your cases array you don't have any entry for `case1` - it is listed as `case` only. This means that this case never actually gets checked.